src/ObjWriting/Game/T6/AssetDumpers/AssetDumperSndBank.cpp

```diff
@@ -96,6 +115,119 @@ namespace
+                                    "grp_alerts",
+                                    "grp_air",
+                                    "grp_bink",
+                                    "grp_announcer",
+                                    ""};
+
+    const std::string CURVES_ENUM[]{"default",
+                                    "defaultmin",
+                                    "allon",
+                                    "alloff",
+                                    "rcurve0",
+                                    "rcurve1",
+                                    "rcurve2",
+                                    "rcurve3",
+                                    "rcurve4",
+                                    "rcurve5",
+                                    "steep",
+                                    "sindelay",
+                                    "cosdelay",
+                                    "sin",
+                                    "cos",
+                                    "rev60",
+                                    "rev65",
+                                    ""};
+
+    const std::unordered_map<unsigned int, std::string> CURVES_MAP{
+        {4135636924, CURVES_ENUM[0] }, // "default"
+        {1298231670, CURVES_ENUM[1] }, // "defaultmin"
+        {2783299419, CURVES_ENUM[2] }, // "allon"
+        {2598309331, CURVES_ENUM[3] }, // "alloff"
+        {2462421902, CURVES_ENUM[4] }, // "rcurve0"
+        {2462421903, CURVES_ENUM[5] }, // "rcurve1"
+        {2462421904, CURVES_ENUM[6] }, // "rcurve2"
+        {2462421905, CURVES_ENUM[7] }, // "rcurve3"
+        {2462421906, CURVES_ENUM[8] }, // "rcurve4"
+        {2462421907, CURVES_ENUM[9] }, // "rcurve5"
+        {3711863914, CURVES_ENUM[10]}, // "steep"
+        {4107033168, CURVES_ENUM[11]}, // "sindelay"
+        {932232097,  CURVES_ENUM[12]}, // "cosdelay"
+        {818663411,  CURVES_ENUM[13]}, // "sin"
+        {686872930,  CURVES_ENUM[14]}, // "cos"
+        {3885755896, CURVES_ENUM[15]}, // "rev60"
+        {3885755901, CURVES_ENUM[16]}, // "rev65"
+    };
+
+    const std::string DUCK_GROUPS_ENUM[]{
+        "snp_alerts_gameplay",
+        "snp_ambience",
+        "snp_claw",
+        "snp_destructible",
+        "snp_dying",
+        "snp_dying_ice",
+        "snp_evt_2d",
+        "snp_explosion",
+        "snp_foley",
+        "snp_grenade",
+        "snp_hdrfx",
+        "snp_igc",
+        "snp_impacts",
+        "snp_menu",
+        "snp_movie",
+        "snp_music",
+        "snp_never_duck",
+        "snp_player_dead",
+        "snp_player_impacts",
+        "snp_scripted_moment",
+        "snp_set_piece",
+        "snp_special",
+        "snp_vehicle",
+        "snp_vehicle_interior",
+        "snp_voice",
+        "snp_weapon_decay_1p",
+        "snp_whizby",
+        "snp_wpn_1p",
+        "snp_wpn_3p",
+        "snp_wpn_turret",
+        "snp_x2",
+        "snp_x3",
+    };
+
+    const std::string LOOP_TYPES_ENUM[]{"nonlooping", "looping"};
```

Comment:
this is unused i think.

`LOAD_TYPES_ENUM` should be as well

Reply:
Removed the loop types. I left LOAD_TYPES for now because I believe that does correspond to one of the columns I'm currently missing. It maps to the flags0, just not sure which column in the file it belongs to.